That FEM test is flaky, it needs to have the fuzzy gmsh comparison code added to it (we've done that already for some others). I've started a re-run to see if that will get the CI green. @FEA-eng and/or @marioalexis84 can you look into cleaning up these tests? 
```
  FAIL: test_GMSHTransfiniteAutomation  — AssertionError: 25 != 91 (node count)
  FAIL: test_GMSHTransfiniteManual      — AssertionError: 22 != 31 (node count)
```

--- a/src/Mod/Part/App/TopoShapeVertexPyImp.cpp
+++ b/src/Mod/Part/App/TopoShapeVertexPyImp.cpp
@@ -54,7 +54,7 @@ std::string TopoShapeVertexPy::representation() const
         str << p.X() << ", " << p.Y() << ", " << p.Z();
         str << ")>";
     }
-    catch (Standard_Failure) {
+    catch (Standard_Failure&) {
         str << "<Part.Vertex()>";
     }
 
@@ -229,8 +229,8 @@ PyObject* TopoShapeVertexPy::richCompare(PyObject* self, PyObject* object, int o
 {
     if (!PyObject_TypeCheck(self, &(TopoShapeVertexPy::Type))
         || !PyObject_TypeCheck(object, &(TopoShapeVertexPy::Type))) {
-        PyErr_SetString(PyExc_TypeError, "both operands must be Vertices");
-        return nullptr;
+        Py_INCREF(Py_NotImplemented);
+        return Py_NotImplemented;
     }
 
     try {

